analysis/webservice/algorithms/doms/ExecutionStatus.py

```diff
@@ -54,7 +54,7 @@ def calc(self, request, **args):
                 )
 
         job_status = NexusExecutionResults.ExecutionStatus(execution_details['status'])
-        host = f'{request.requestHandler.request.protocol}://{request.requestHandler.request.host}'
+        host = f'https://{request.requestHandler.request.host}'
```

Comment:
What's the reason for this change?

Reply:
@skorper This was an issue with CDMS. The links for NetCDF and CSV result downloads were being blocked by Chrome or something, and when they were downloaded with right click and save as they were quarantined or something requiring additional confirmation to be able to be used.
We noticed all the links were `http://` links and figured it was a security feature to distrust downloads from unsecured URLs so I forced the links to be `https://`.